File: classdata.py
import pandas as pd
import seaborn as sns
import matplotlib.pyplot as plt

data1 = pd.read_csv("/Users/jonathanswap/Desktop/MSDS summer semester/MSDS-Orientation-Computer-Survey(in).csv")

sns.histplot(data1['RAM (in GB)'], bins=30, color='orange', alpha=0.5)
plt.title("Count of RAM (in GB) in the current and past MSDS cohorts")
plt.xlabel("RAM (in GB)")
plt.ylabel("Count")
plt.show()

File: classdata.R
library(ggplot2)
data1 <- read_xlsx("MSDS-Orientation-Computer-Survey.xlsx")




f <- ggplot(data = data1, aes(x = `RAM (in GB)`))
+ geom_histogram(fill = "blue", alpha = .5) +
  ggtitle("Count of RAM (in GB) in the current and past MSDS cohorts")
f
